packages/typespec-ts/src/index.ts

```diff
@@ -89,7 +89,7 @@ export async function $onEmit(context: EmitContext) {
     compilerContext: context,
     tcgcContext: dpgContext
   });
-  provideSdkTypes(dpgContext.experimental_sdkPackage);
+  provideSdkTypes(dpgContext.sdkPackage);
```

Comment:
This is tcgc change https://azure.github.io/typespec-azure/docs/next/release-notes/release-2024-07-16#azure-toolstypespec-client-generator-core